kii-core/kii_core.c

```diff
@@ -1584,10 +1584,12 @@ kii_core_api_call_start(
         return KIIE_FAIL;
     }
 
-    if (prv_kii_http_set_header(kii, "content-type",
-                    content_type) != KII_HTTPC_OK) {
-        M_KII_LOG(M_REQUEST_HEADER_CB_FAILED);
-        return KIIE_FAIL;
+    if (content_type != NULL) {
```

Comment:
How can we set content_type == NULL? Can we specify NULL as content_type parameter in kii_api_call_start()? If so, please describe it in the Doxygen document.